packages/components/src/styles/paragraph/paragraph.stories.ts

```diff
@@ -41,22 +39,41 @@ export const Default = {
   }
 };
 
+/**
+ * Use the `sd-paragraph--size-` class to set the size:
+ * - `sd-paragraph--size-default`
```

Comment:
We need to create a ticket to fix the name of this variant as in FIGMA the default is named as Large

Reply:
There was a mistake, default doesn't exists. `sd-paragraph--size-default` will delete.

Reply:
Done